workspaces/theme/plugins/theme/src/lightTheme.ts

```diff
@@ -78,7 +78,8 @@ export const lightThemeOverrides: Partial<ThemeConfigPalette> = {
 
       contrastText: '#FFF',
 
-      appBarBackgroundColor: '#212427',
+      appBarBackgroundColor: '#f2f2f2',
```

Comment:
Is this background color change for matching PF6 style?

Reply:
here's an after picture in dark mode (after some global header changes are made)

![image](https://github.com/user-attachments/assets/926434aa-9edd-4377-91ff-fa4107648577)

Reply:
Awesome! It looks nice in dark mode. Can you include both screenshots for light mode and dark mode in the PR description and check the last item in the checklist?

Also to track this effort, feel free to create a jira ticket and pull into our Sprint. And if there's an epic for matching styles with PF6, link this ticket to that epic.